sky/clouds/service_catalog/scp_catalog.py

```diff
@@ -67,7 +67,7 @@ def get_default_instance_type(
 
 
 def get_accelerators_from_instance_type(
-        instance_type: str) -> Optional[Dict[str, int]]:
+        instance_type: str) -> Optional[Dict[str, Union[int, float]]]:
```

Comment:
For the clouds that won't return `float`, we probably don't need to update it to `Union[int, float]`?

Reply:
Same, here is what I got:

```bash
./format.sh
SkyPilot Black:
All done! ✨ 🍰 ✨
4 files left unchanged.
SkyPilot yapf: Done
SkyPilot isort:
Skipped 5 files
SkyPilot mypy:
sky/clouds/service_catalog/aws_catalog.py:248: error: Incompatible return value type (got "Optional[Dict[str, Union[int, float]]]", expected "Optional[Dict[str, int]]")  [return-value]
Found 1 error in 1 file (checked 225 source files)
```